src/main/python/gcloud_hydra/mesos_marathon_setup.py

```diff
@@ -0,0 +1,235 @@
+# Zookeeper to keep track of the current leader of the master servers.
```

Comment:
I would suggest to integrate the setup script with the hydra command line.
Something like "hydra setup" should launch this script.